docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,615 @@
+---
+title: "Use ML and a webcam to entertain kids with bedtime songs"
+linkTitle: "Bedtime Songs Bot"
+weight: 60
+type: "docs"
+tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
+description: "Create a robot babysitter with a webcam and machine learning."
+image: "/tutorials/bedtime-songs-bot/bedtime-songs-bot.png"
+imageAlt: "Tess holds up brightly colored puzzle pieces in front of the camera of a Macbook laptop."
+images: ["/tutorials/bedtime-songs-bot/bedtime-songs-bot.png"]
+authors: [ "Tess Avitabile" ]
+languages: [ "python" ]
+viamresources: [ "camera", "sensor", "mlmodel", "vision" ]
+level: "Beginner"
+date: "21 April 2023"
+# cost: 0 (laptop)
```

Comment:
```suggestion
cost: "0"
```

Reply:
Thank you for taking the time to comment this inline I appreciate it on my end haha, it makes the editing process super quick.